Dockerfile-local

```diff
@@ -14,8 +14,10 @@
 # limitations under the License.
 
 ARG CP_VERSION
+ARG NAMESPACE=confluentinc 
+ARG IMAGE=cp-kafka-connect
 
-FROM confluentinc/cp-kafka-connect:$CP_VERSION
+FROM confluentinc/$IMAGE:$CP_VERSION
```

Comment:
1.  Here this is hardcoded to `confluentinc` and above it's `BASE_PREFIX`.  Is there a reason for them to diverge?  Should they be consistent?

2. To disambiguate, what if we rename `IMAGE` to `CONNECT_IMAGE`?

Reply:
1. Good catch, thank you
2. Yep, done